stylesheets/partials/features/_header-main-menu.scss

```diff
@@ -29,6 +29,11 @@
 		font-size: 1.5em;
 	}
 	.nav-and-search {
+		
+		button:focus {
+			outline-color: transparent;	
+		}
```

Comment:
You should talk to Dan and Sam about this if you haven't yet. We'll need some sort of focus indication for keyboard-only users.

Reply:
ok, ill talk to dan before merging

Reply:
added back, looks good confirmed by @danfox01